migrations/1571079562023_add-is-public-to-projects.ts

```diff
@@ -0,0 +1,20 @@
+import { MigrationBuilder } from 'node-pg-migrate'
+import { Project } from '../src/Project'
+
+const tableName = Project.tableName
+const columnName = 'is_public'
+const columns = {
+  [columnName]: {
+    type: 'BOOLEAN',
+    default: false,
+    notNull: true
+  }
+}
+
+export const up = (pgm: MigrationBuilder) => {
+  pgm.addColumn(tableName, columns)
+}
+
+export const down = (pgm: MigrationBuilder) => {
+  pgm.dropColumn(tableName, columnName)
```

Comment:
This is OK but in case you didn't know, the `down` method can be safely inferred by pgm using the `up` command (meaning that it's not necessary)

Reply:
👍